src/Controller/FightController.php

```diff
@@ -26,6 +26,28 @@ public function index(): string
         return $this->twig->render('Fight/index.html.twig', ['fights' => $fights]);
     }
 
+    public function initiateFighter()
+    {
+        $fighterManager = new FighterManager();
+
+        $fighter1AsArray = $fighterManager->selectOneById(1);
+        $fighter2AsArray = $fighterManager->selectOneById(2);
```

Comment:
try to configure your manager so you can fetch classes ;)

https://www.php.net/manual/fr/pdostatement.setfetchmode.php